src/Command/DefaultProcessFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Command;
+
+use Symfony\Component\Process\Process;
+
+final class DefaultProcessFactory implements ProcessFactory
```

Comment:
DefaultProcessFactory doesn't tell much about what the implementation is, something like SymfonyProcessFactory would be more informative.